examples/sst/stacks/AppRouter.ts

```diff
@@ -7,6 +7,9 @@ export function AppRouter({ stack }) {
     bind: [],
     environment: {},
     timeout: "20 seconds",
+    experimental: {
```

Comment:
We can use the default `NextJsSite` here, no need to override anything anymore

Reply:
Oh yea good call, updating...